conans/client/generators/scons.py

```diff
@@ -0,0 +1,37 @@
+from conans.model import Generator
+
+class SConsGenerator(Generator):
+    @property
+    def filename(self):
+        return "SConscript-conan"
```

Comment:
The man page says "By convention, these subsidiary files are named SConscript, although any name may be used".
It doesn't really matter.